immuni_exposure_ingestion/models/upload.py

```diff
@@ -32,7 +32,7 @@ class Upload(Document):
     """
 
     to_publish = BooleanField(default=True)
-    keys = EmbeddedDocumentListField(TemporaryExposureKey, required=True)
+    keys = EmbeddedDocumentListField(TemporaryExposureKey, required=False, default=[])
```

Comment:
🤔 there's a difference between having it not required and having it required as an empty array, can't we do the latter?

Reply:
There is indeed a difference but, unless I am missing some info, setting an empty list into a required EmbeddedDocumentList field raises a `ValidationError: field is required and cannot be empty`